.github/CODEOWNERS

```diff
@@ -6,5 +6,5 @@
 *freebsd*	@markjdb @tuexen
 *netbsd*	@krytarowski @R3x
 *openbsd*	@blackgnezdo @mptre
```

Comment:
GH also shows warning for blackgnezdo ("make sure it exists and have write access").

Reply:
That is a more complex question. User exists, but access rights are "Triage", not "Write".
Here the situation is easier. Lets go 1 by 1.